lib/ReadsAlignmentUtils/ReadsAlignmentUtilsImpl.py

```diff
@@ -188,7 +191,52 @@ def _get_aligner_stats(self, bam_file):
         Gets the aligner stats from BAM file
         """
         path, file = os.path.split(bam_file)
-        return self.samtools.get_stats(file, path)
+
+        self.__LOGGER.info('Start to generate aligner stats')
+        start_time = time.time()
+
+        infile = pysam.AlignmentFile(bam_file, 'r')
```

Comment:
do you need to close this file?

Reply:
@jayrbolton Oh no, I didn't close the file. Nice catch. Thanks!